packages/@react-types/tag/src/index.d.ts

```diff
@@ -21,7 +21,7 @@ export interface TagGroupProps<T> extends Omit<CollectionBase<T>, 'disabledKeys'
   onRemove?: (key: Key) => void
 }
 
-export interface SpectrumTagGroupProps<T> extends TagGroupProps<T>, DOMProps, StyleProps {}
+export interface SpectrumTagGroupProps<T> extends TagGroupProps<T>, DOMProps, StyleProps, AriaLabelingProps {}
```

Comment:
A bit strange IMO that the `AriaLabelingProps` are extended here instead of on something like `AriaTagGroupProps` but we can discuss/address in future iterations since that would involve moving some hooks around (e.g. should useGrid be in useTagGroup? I forget if we already made a decision about that before...).

Reply:
To be done later